docs/README-AWS.md

```diff
@@ -398,70 +479,99 @@ inputs:
           es_datastream_name: "logs-generic-default"
 ```
 
-#### Notes
-`inputs.[].expand_event_list_from_field` can be defined as string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
+**Notes:**
+
+`inputs.[].expand_event_list_from_field` can be defined as a string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
+
+You should note that when relying on the [Routing support for AWS Service Logs](#routing-support-for-aws-services-logs), any value set for the `expand_event_list_from_field` configuration parameter will be ignored, because this will be automatically handled by the Elastic Serverless Forwarder.
+
+### Example
 
-Beware that when relying on the [Routing support for AWS Service Logs](#routing-support-for-aws-services-logs), any value set for `expand_event_list_from_field` configuration param will be ignored, since it will be taken care automatically by the Elastic Serverless Forwarder.
+With the following input:
 
-#### Example:
-Having the following content from the input:
 ```json lines
 {"Records":[{"key": "value #1"},{"key": "value #2"}]}
 {"Records":[{"key": "value #3"},{"key": "value #4"}]}
 ```
-- without setting `expand_event_list_from_field` two events will be forwarded:
+
+Without setting `expand_event_list_from_field`, two events will be forwarded:
+
     ```json lines
     {"@timestamp": "2022-06-16T04:06:03.064Z", "message": "{\"Records\":[{\"key\": \"value #1\"},{\"key\": \"value #2\"}]}"}
     {"@timestamp": "2022-06-16T04:06:13.888Z", "message": "{\"Records\":[{\"key\": \"value #3\"},{\"key\": \"value #4\"}]}"}
     ```
 
-- if `expand_event_list_from_field` value is set to `Records`, four events will be forwaderd:
+If `expand_event_list_from_field` is set to `Records`, four events will be forwarded:
+
     ```json lines
     {"@timestamp": "2022-06-16T04:06:21.105Z", "message": "{\"key\": \"value #1\"}"}
     {"@timestamp": "2022-06-16T04:06:27.204Z", "message": "{\"key\": \"value #2\"}"}
     {"@timestamp": "2022-06-16T04:06:31.154Z", "message": "{\"key\": \"value #3\"}"}
     {"@timestamp": "2022-06-16T04:06:36.189Z", "message": "{\"key\": \"value #4\"}"}
     ```
 
+## Routing AWS Service Logs
+
+For `S3 SQS Event Notifications input` the Lambda function supports automatic routing of several AWS service logs to the corresponding [integration data streams](https://docs.elastic.co/en/integrations) for further processing and storage in the Elasticsearch cluster.
```

Comment:
`S3 SQS Event Notifications input` is reworded as `S3 SQS Event Notifications` at https://github.com/elastic/elastic-serverless-forwarder/pull/131/files#diff-e9ecc3d1058b2f97b23c473ba67681d3e8e5c20546a6a7157b3bf9c1fc3c2e9cR53